coralproject/talk
# [CORL-864] Change load amounts for comments

## What does this PR do?

Load 20 comments initially and on load more instead of 10.
Load 3 replies initially and on load more instead of 5 deep.

## How do I test this PR?

- Check that initial comment load is 20.
- Check that it can load 20 more on "Load More".
- Check that replies load 3 deep initially.
- Check that replies show "Read more of this conversation" after 3 replies deep.
